pkgs/development/compilers/mrustc/patches/0001-dont-download-rustc.patch

```diff
@@ -0,0 +1,11 @@
+--- a/minicargo.mk	2023-04-06 08:26:18.408817572 +0200
++++ b/minicargo.mk	2023-04-06 08:27:11.553536996 +0200
+@@ -176,7 +176,7 @@
+ 	@echo [CURL] $@
+ 	@rm -f $@
+ 	@curl -sS https://static.rust-lang.org/dist/$@ -o $@
+-$(RUSTC_SRC_DL): $(RUSTC_SRC_TARBALL) rustc-$(RUSTC_VERSION)-src.patch
++$(RUSTC_SRC_DL): rustc-$(RUSTC_VERSION)-src.patch
```

Comment:
I don't get why this changes anything. The `$(RUSTC_SRC_DL)` file already exists, but for some reason Make wants to rebuild this target when it has `$(RUSTC_SRC_TARBALL)` as a dependency.

Reply:
Ideally this comment/information should end in the header of this patch somewhat